### DISCUSSION TITLE
Can't log in: Failed to fetch user

### DISCUSSION THREAD

**Discussion by NadCat1** [SevenTV/chatterino7#169] [Q&A] [answered]
Hi, whenever I try to login using the browser it just returns me to https://chatterino.com/client_login and it says "Error: Failed to fetch user". I've tried it with different browsers and with and without a VPN (and after restarting my PC). I would try using the other login methods but I can't find anything about where to find my User ID, Client ID, and which OAuth token is being asked for. Is there anything I need to look out for or enable in my Twitch account to be able to log in?

**Comment by Felanbird** [comment, marked as answer]
F12 and go to Console and take a 👀 while you are trying to login, but it's very likely Twitch Helix is just having an issue

**Reply by NadCat1** [reply]
It says "Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://api.twitch.tv/helix/users. (Reason: CORS request did not succeed). Status code: (null).". I've found a few extensions that claim to enable CORS but on this [testpage](https://webbrowsertools.com/test-cors/) FETCH :: GET (credentials) is still blocked and I still can't log in

**Reply by Excellify** [reply]
What browser do you use and do you use any weird ad blockers?

**Reply by NadCat1** [reply]
Weird, I tried doing it in a private window in Firefox (so no addons) and it didn't work. I tried it again in a private window in Waterfox and that did work

**Reply by Felanbird** [reply]
> (so no addons) 

Private window does not guarantee no addons, there's a setting to allow an addon to run in private browsing.

**Reply by NadCat1** [reply]
Oh. I did indeed have a weird ad blocker allowed in private windows for some reason...
